.env.sample

```
@@ -1,5 +1,8 @@
-# requires full url
-NEXT_PUBLIC_BACKEND_APP_URL = 'http://0.0.0.0:8000/api/...'
+# client app public URL (used in og:image and og:url metadata)
+NEXT_PUBLIC_CLIENT_DOMAIN =
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)